interface/login/login.php

```diff
@@ -233,6 +255,13 @@ function imsubmitted() {
 <a  href="../../acknowledge_license_cert.html" target="main"><?php echo xlt('Acknowledgments, Licensing and Certification'); ?></a>
 </div>
 </div>
+
+<div class="product-registration-modal" style="display: none">
+  <p class="context"><?php echo xlt("Register your installation with OEMR 501(c)(3) to recieve implement update notifications."); ?></p>
```

Comment:
Could stress the security such as:
Register your installation with OEMR 501(c)(3) to receive important notifications, such as security fixes and new release announcements.